shlr/rizin-shell-parser/grammar.js

```diff
@@ -1,658 +1,728 @@
-            optional(field('args', $.args)),
-        )),
-        _interpret_command: $ => prec.left(1, choice(
-            seq(
-                field('command', alias('.', $.cmd_identifier)),
-                field('args', $._simple_command),
-            ),
-            seq(
-                field('command', alias(/\.[\.:\-*]+/, $.cmd_identifier)),
-                /[ ]+/,
-                field('args', optional($.args)),
-            ),
-            seq(
-                field('command', alias(/\.[ ]+/, $.cmd_identifier)),
-                field('args', optional($.args)),
-            ),
-            seq(
-                field('command', alias('.!', $.cmd_identifier)),
-                field('args', $.interpret_arg),
-            ),
-            seq(
-                field('command', alias('.(', $.cmd_identifier)),
-                field('args', $.macro_call_content),
-            ),
-            seq(
-                field('command', alias($._interpret_search_identifier, $.cmd_identifier)),
-                field('args', $.args),
-            ),
-            prec.right(1, seq(
-                field('args', $._simple_command),
-                field('command', '|.'),
-            )),
-        )),
-        _interpret_search_identifier: $ => seq('./'),
-        _pf_arged_command: $ => choice(
-            seq(
-                field('command', alias($.pf_dot_cmd_identifier, $.cmd_identifier)),
+            optional(alias(/[$]+/, $.grep_specifier_identifier))
+          ),
+          alias(/[$]+/, $.grep_specifier_identifier)
+        )
+      ),
+
+    html_disable_command: ($) =>
+      prec.right(1, seq(field("command", $._simple_command), "|")),
+    html_enable_command: ($) =>
+      prec.right(1, seq(field("command", $._simple_command), "|H")),
+    pipe_command: ($) => seq($._simple_command, "|", $.pipe_second_command),
+    pipe_second_command: ($) => /[^|\r\n;]+/,
+
+    _offsetsizes_args: ($) => repeat1(seq($.arg, $.arg)),
+
+    iter_file_lines_command: ($) =>
+      prec.right(1, seq($._simple_command, "@@.", $.arg)),
+    iter_offsets_command: ($) =>
+      prec.right(1, seq($._simple_command, "@@=", optional($.args))),
+    iter_offsetssizes_command: ($) =>
+      prec.right(
+        1,
+        seq(
+          $._simple_command,
+          "@@@=",
+          optional(alias($._offsetsizes_args, $.args))
+        )
+      ),
+    iter_hit_command: ($) =>
+      prec.right(
+        1,
+        seq(
+          $._simple_command,
+          "@@",
+          $._concat,
+          alias($._search_command, $.arged_command)
+        )
+      ),
+    iter_interpret_command: ($) =>
+      prec.right(1, seq($._simple_command, "@@c:", $._simple_command)),
+    iter_interpret_offsetssizes_command: ($) =>
+      prec.right(1, seq($._simple_command, "@@@c:", $._simple_command)),
+    iter_comment_command: ($) =>
+      prec.right(1, seq($._simple_command, "@@C", optional(seq(":", $.arg)))),
+    iter_dbta_command: ($) =>
```

Comment:
what is the max width? I think it should be increased a bit... Having these lines differently is a bit confusing.

Reply:
80
I can change them to 120 if you want.

Reply:
yes please. at least 100.

Reply:
I move it to 120 to simplify lambda declaration.